graalpy-embedded-pillow/pom.xml

```diff
@@ -0,0 +1,177 @@
+ the Larger Works (as defined below), to deal in both
+
+ (a) the Software, and
+
+ (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ one is included with the Software each a "Larger Work" to which the Software
+ is contributed by such licensors),
+
+ without restriction, including without limitation the rights to copy, create
+ derivative works of, display, perform, and distribute the Software and make,
+ use, sell, offer for sale, import, export, have made, and have sold the
+ Software and the Larger Work(s), and to sublicense the foregoing rights on
+ either these or other terms.
+
+ This license is subject to the following condition:
+
+ The above copyright notice and either this complete permission notice or at a
+ minimum a reference to the UPL must be included in all copies or substantial
+ portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.example</groupId>
+  <artifactId>embedded-pillow</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- Select the GraalPy version to use. -->
+    <graalpy.version>24.0.1</graalpy.version>
+    <graalpy-plugin.version>24.0.0</graalpy-plugin.version>
+    <main.class>com.oracle.example.graalpy.PillowFrame</main.class>
+    <graalpy.edition>python</graalpy.edition>
+    <maven.compiler.release>21</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <nativeimage.agentlib.argument>-showversion</nativeimage.agentlib.argument>
+  </properties>
+
+  <dependencies>
+    <!-- The Python dependency is always required  -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>${graalpy.edition}</artifactId>
+      <version>${graalpy.version}</version>
+      <type>pom</type>
+    </dependency>
+    <!-- The dependency on python-launcher is only required if the graalpy
+         plugin is used with packages -->
+    <dependency>
+      <groupId>org.graalvm.python</groupId>
+      <artifactId>python-launcher</artifactId>
+      <version>${graalpy.version}</version>
+    </dependency>
+    <!-- This dependency is required if packages should be embedded in
+         the Jar.  -->
```

Comment:
<!-- This dependency is required if packages should be embedded in  the JAR file.  -->